WcaOnRails/config/routes.rb

```diff
@@ -20,6 +20,7 @@
   end
   post 'registration/:id/refund/:payment_id' => 'registrations#refund_payment', as: :registration_payment_refund
   resources :users, only: [:index, :edit, :update]
+  get 'users/sign_up' => 'users#sign_up'
```

Comment:
What happened here?

Reply:
I needed a route to the /users/sign_up page to link it in the public pages (info and register). As I couldn't see it anywhere. I created it. Is this wrong/did I miss something really obvious?

Reply:
I'm pretty sure we already have this. See:

```
bin/rake routes | grep sign_up
                            new_user_registration GET    /users/sign_up(.:format)                                                                 accounts/registrations#new
```

You can see we use `new_user_registration_path` in a number of places, such as https://github.com/jfly/worldcubeassociation.org/blob/daa2b91fcdf1797cd204b2574d09cc9e276c3776/WcaOnRails/app/views/static_pages/faq.html.erb#L10.